### ISSUE TITLE
Failed to fetch but Response Code 200

### ISSUE THREAD

**Issue by ghost** [getinsomnia/insomnia#5001]
### Expected Behavior

To get the Response Body 

### Actual Behavior

No Response Body

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

Insomnia (not working)
![image](https://user-images.githubusercontent.com/6480047/180595446-3d5966bc-da6a-403f-adbe-cbafd7887410.png)

Postman (working)
![image](https://user-images.githubusercontent.com/6480047/180595495-67f59b8f-cb89-43cd-9f81-ff0f80e6c4c2.png)

Test yourself: https://codefirstrestapi-docker.azurewebsites.net/index.html POST: api/auth

### Insomnia Version

2022.4.2.

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### Installation method

download windows installer Insomnia.Core-2022.4.2.exe

### Last Known Working Insomnia version

2022.4.2.

**Comment by filfreire** [comment]
Hi @ericbrunner, thanks for reporting this!

I wasn't able to reproduce this by calling the request using latest insomnia (same version as the one you use):

![image](https://user-images.githubusercontent.com/11976836/181017305-89193220-29f9-4eca-a20f-51214ff0960b.png)

Can you share more details of the failure, and perhaps share the contents of the Timeline:

![Screenshot_2022-07-26_at_14_28_01](https://user-images.githubusercontent.com/11976836/181017592-ff3ddb57-9ba1-44aa-9d4a-e7ba43b30305.png)

**Comment by ghost** [comment]
Hi @filfreire

My screenshot looks totally different. I Imported the swagger.json

![image](https://user-images.githubusercontent.com/6480047/181026844-8db50863-8625-4635-9c40-061a9b006847.png)
 

Direct HTTP Post Request works as on your side. So there is an issue with OpenAPI Import
![image](https://user-images.githubusercontent.com/6480047/181027692-e60e6eb2-adaa-4f0d-8072-6c75253efd28.png)

**Comment by filfreire** [comment]
@ericbrunner I see, seems it's some problem with the Swagger preview part, I missed that from when I first looked at the issue.

Would you mind sharing that example swagger file here where you are able to reproduce this?

**Comment by ghost** [comment]
https://codefirstrestapi-docker.azurewebsites.net/swagger/v1/swagger.json

**Comment by filfreire** [comment]
@ericbrunner I found out the issue, you were missing the `servers` part in your spec, which meant that the curl when being sent from that document preview had an incomplete URL (not pointing to any server).

Once I added:
```
"servers": [
    {
      "url": "https://codefirstrestapi-docker.azurewebsites.net"
    }
  ]
```

to the specification, I was able to reach your server and get responses back in the preview from the server:

![Image](https://user-images.githubusercontent.com/11976836/184342115-6d451720-f314-4002-b567-202dd2af5df0.png)

I'm going to go ahead and mark this issue as closed, but let me know if this helped. Feel free to report any other issues or improvements you come across.
